README.md

```diff
@@ -65,6 +65,39 @@ customElements.define('new-element', NewElement);
 
 See examples in the demo or try this live [JS bin](https://jsbin.com/zitotejimi/edit?html,output).
 
+## Uprading from 3.x
+
+The component has been migrated to LitElement and uses TypeScript now. This migration introduced two breaking changes.
+
+### Removed Polymer-specific `selection-changed` event used for 2-way bindings
```

Comment:
"Removed Polymer-specific `selection-changed` event"

We can explain about its usage in the paragraph below.